bigquery/tests/unit/test__pandas_helpers.py

```diff
@@ -31,7 +31,7 @@
     import pyarrow
     import pyarrow.types
 except ImportError:  # pragma: NO COVER
-    pyarrow = None
+    pyarrow = mock.Mock()
```

Comment:
Why was this needed? `None` causing problems somewhere?

Reply:
Yes, it was causing problems in the test parameterization. I've added a comment about this.

https://github.com/googleapis/google-cloud-python/blob/0588f19c40cf921c4975afd63f639a47c271da4e/bigquery/tests/unit/test__pandas_helpers.py#L110-L127